_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,266 @@
+	<br />
+	<label for="address">Address</label>
+	<input type="text" id="address" />
+	<p>Pick a color (required)</p>
+	<label><input type="radio" name="color" value="blue" required />Blue</label>
+	<label><input type="radio" name="color" value="yellow" />Yellow</label>
+	<br />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The `input` element does not have a [form field error indicator][].
+
+```html
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<input type="button" value="Submit" />
+</form>
+```
+
+#### Failed Example 2
+
+The multiple `input` elements share a [form field error indicator][] but its message does not identify the elements that caused the error nor describes the cause of the error.
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		if (!age || isNaN(age) || age < 1) {
+			document.getElementById('error').innerText = 'Please fill the field correctly.'
+		}
+		var name = document.getElementById('name').value
+		if (!name) {
+			document.getElementById('error').innerText = 'Please fill the field correctly.'
+		}
+	}
+</script>
+
+<form>
+	<div id="error"></div>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<label for="name">Name</label>
+	<input type="text" id="name" />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+#### Failed Example 3
+
+The `input` elements has a [form field error indicator][] but its message does not describe the cause of the error.
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		if (!age || isNaN(age) || age < 1) {
+			document.getElementById('error').innerText = 'Please enter a correct age.'
+		}
+	}
+</script>
+
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<span id="error"></span><br />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+#### Failed Example 4
+
+The `input` element has a [form field error indicator][] that identifies it (by referencing its label) and describes the cause of the error but the message is not [visible][].
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		console.log(isNaN(age))
+		if (!age || isNaN(age)) {
+			document.getElementById('error').innerText = 'Age must be a number'
+		} else if (age < 1) {
+			document.getElementById('error').innerText = 'Age must be at least 1'
+		}
+	}
+</script>
+
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<span id="error" style="position: absolute; top: -9999px; left: -9999px;"></span><br />
+	<input type="button" value="Submit" onclick="processForm()" />
```

Comment:
```suggestion
	<input type="button" value="Submit" onclick="processForm()" aria-describedby="error" />
```

This will still work with display:none.